### DISCUSSION TITLE
Outreachy final plan

### DISCUSSION THREAD

**Discussion by esraa-abdelmaksoud** [ModECI/MDF#243] [Q&A] [answered]
Hello @pgleeson,

I can see while finalizing Outreachy's application that I should discuss a final plan for the internship. How do you think we can sort this out together before the deadline? Thanks! :)

**Comment by pgleeson** [comment, marked as answer]
@esraa-abdelmaksoud Do you mean the timeline? What I'd like to see here is your idea on what needs to be done over the period of the project, broken into ~2 week blocks. Doesn't have to be very detailed, it will just give us an impression of how you plan to manage your time, and what steps you think are important to achieve the goals of the project.
